fleetspeak/client-mac/build.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
+
+realpath() {
+    [[ $1 = /* ]] && echo "$1" || echo "$PWD/${1#./}"
+}
+
+CONFIGS_DIR=$(pwd)
+FS_BINARY=$(realpath $1)
+FS_VERSION=$(cat ../../VERSION)
+COMPONENT_PKG_NAME="fleetspeak-client-component.xar"
+PRODUCT_ARCHIVE_NAME="fleetspeak-client-${FS_VERSION}.pkg"
+
+rm -rf ./work
+mkdir work
+cd work
+
+mkdir -p pkg_root/Library/LaunchDaemons
+mkdir -p pkg_root/etc/fleetspeak/services
+mkdir -p pkg_root/etc/fleetspeak/textservices
```

Comment:
I think we should use /etc/fleetspeak-client to store configs (the writeback file with the client's identity will live there as well).

Reply:
Done.